.env

```
@@ -1,8 +1,13 @@
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
-
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+# You should copy this file to .env.local and fill it in for your environment
 
 DATABASE_URL="file:./dev.db"
-OLLAMA_MODEL="mistral"
\ No newline at end of file
+
+OLLAMA_MODEL="mistral"
+
+AUTH_SECRET="f00b4r"              # run `pnpm dlx auth secret to generate`
```

Is this supposed to be
```suggestion
AUTH_SECRET="f00b4r"              # run `pnpm dlx auth secret` to generate
```
?